File: app/src/main/java/org/evcs/android/model/Session.java
package org.evcs.android.model;

import org.joda.time.DateTime;

import java.io.Serializable;

public class Session extends Charge implements Serializable {
    private int id;
//    "kwh":"0.0",
//    "status":"active",
    private DateTime updatedAt;
    private int stationName;
//duration

    public int getId() {
        return id;
    }

    public DateTime getUpdatedAt() {
        return updatedAt;
    }

    public int getStationName() {
        return stationName;
    }

}

File: app/src/main/java/org/evcs/android/model/Session.kt
package org.evcs.android.model

import org.joda.time.DateTime
import java.io.Serializable

class Session : Charge(), Serializable {
    //duration
    //    "kwh":"0.0",
    //    "status":"active",
    val updatedAt: DateTime? = null
    val ongoingRate: OngoingRate? = null
    val firstSoc: Int? = null
    val lastSoc: Int? = null
}